infra/src/test/kotlin/fi/fta/geoviite/infra/tracklayout/TrackLayoutDomainTestData.kt

```diff
@@ -976,6 +977,15 @@ fun externalIdForTrackNumber(): Oid<LayoutTrackNumber> {
     return Oid("$first.$second.$third.$fourth")
 }
 
+fun externalIdForSwitch(): Oid<LayoutSwitch> {
+    val first = nextInt(100, 999)
+    val second = nextInt(100, 999)
+    val third = nextInt(100, 999)
+    val fourth = nextInt(100, 999)
+
+    return Oid("$first.$second.$third.$fourth")
+}
```

Comment:
Tämä tuli uutena, kaikki muut muutokset ovat formatteria. Tämä on käytännössä duplikointia ylempää. En lähtenyt selvittelemään `nextInt()`:in sielunelämää, vaan päätin toteuttaa tämän niinkuin aiemmatkin. Kyseessä lienee kuitenkin jonkinlainen tilallinen funktio? Sen vuoksi en ukaltanut lähteä refaktoroimaan näitä kaikkia käyttämään jotain yhtä ja samaa pohjatoteutusta